lib/tasks/release.rake

```diff
@@ -24,16 +24,11 @@ task :release do
   lock_release = root.join("Gemfile.lock.release")
   if lock_release.exist?
     gemfile_lock = lock_release.to_s.chomp(".release")
-    appliance_dependency = root.join("bundler.d/manageiq-appliance-dependencies.rb")
```

Comment:
There's more to delete down on line 199

Reply:
And I also think there's stuff in manageiq-rpm_build that will have to be removed.